src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -399,6 +399,7 @@ AlgorithmPaintToolBox::AlgorithmPaintToolBox(QWidget *parent ) :
 
     m_applyButton = new QPushButton( tr("Save") );
     m_applyButton->setToolTip(tr("Save result to the Database"));
+    m_applyButton->setObjectName("saveButton");
```

Comment:
I know we've already had this discussion but I'll explain why I find naming button objects as their visual text:
1. There shouldn't be a dependency between the text displayed by the GUI, and the naming of the object, because those are two separate concerns. If we decide to change the button text to "Apply", we won't be renaming the object as a consequence because you would have no easy way of finding all the occurrences in the code that call findObject with the current name.
2. It only works for things like buttons that have text displayed on them. How do you name widgets that have no text displayed on them, or labels that have a long text (like instructions) ? You would have to use a different naming scheme. I prefer having one consistent scheme for all widgets.
3. The idea of naming buttons by their visual text might seem like a way to counter the problem of having to look into the source code to know what the name of the object is, but like I mentioned in point 2, this will not apply to all cases. If we are doing it this way, it's not going to be clean anyways.